### PR TITLE
[Database Monitoring] Update sqlserver setup documentation to mention log shipping

### PR DESCRIPTION
**Pull request by joelmarcotte**
### What does this PR do? What is the motivation?
- Adds documentation about log shipping monitoring for the SQLServer integration, which will be available in agent 7.50+.

### Merge instructions

- [ ] Please merge after reviewing

### CODE REVIEW

**Review by ynnadkrap — APPROVED** [review]

File: content/en/database_monitoring/setup_sql_server/azure.md [hunk 1]
```diff
@@ -46,6 +46,10 @@ CREATE LOGIN datadog WITH PASSWORD = '<PASSWORD>';
 CREATE USER datadog FOR LOGIN datadog;
 ALTER SERVER ROLE ##MS_ServerStateReader## ADD MEMBER datadog;
 ALTER SERVER ROLE ##MS_DefinitionReader## ADD MEMBER datadog;
+-- If you intend on using monitoring log shipping (availably in agent v7.50+), proceed with the following:
```
Comment (cswatt) [inline content/en/database_monitoring/setup_sql_server/azure.md:49]:
```suggestion
-- To use Log Shipping Monitoring (available in Agent v7.50+), uncomment the next three lines:
```

File: content/en/database_monitoring/setup_sql_server/azure.md [hunk 2]
```diff
@@ -101,6 +105,10 @@ CREATE USER datadog FOR LOGIN datadog;
 GRANT CONNECT ANY DATABASE to datadog;
 GRANT VIEW SERVER STATE to datadog;
 GRANT VIEW ANY DEFINITION to datadog;
+-- If you intend on using monitoring log shipping (availably in agent v7.50+), proceed with the following:
```
Comment (cswatt) [inline content/en/database_monitoring/setup_sql_server/azure.md:108]:
```suggestion
-- To use Log Shipping Monitoring (available in Agent v7.50+), uncomment the next three lines:
```

File: content/en/database_monitoring/setup_sql_server/azure.md [hunk 3]
```diff
@@ -110,6 +118,10 @@ CREATE LOGIN datadog WITH PASSWORD = '<PASSWORD>';
 CREATE USER datadog FOR LOGIN datadog;
 GRANT VIEW SERVER STATE to datadog;
 GRANT VIEW ANY DEFINITION to datadog;
+-- If you intend on using monitoring log shipping (availably in agent v7.50+), proceed with the following:
```
Comment (cswatt) [inline content/en/database_monitoring/setup_sql_server/azure.md:121]:
```suggestion
-- To use Log Shipping Monitoring (available in Agent v7.50+), uncomment the next three lines:
```

File: content/en/database_monitoring/setup_sql_server/rds.md [hunk 4]
```diff
@@ -21,10 +21,13 @@ Database Monitoring provides deep visibility into your Microsoft SQL Server data
...
-1. [Grant the Agent access to the database](#grant-the-agent-access)
-1. [Install the Agent](#install-the-agent)
-1. [Install the RDS integration](#install-the-rds-integration)
+- [Before you begin](#before-you-begin)
+- [Configure the AWS integration](#configure-the-aws-integration)
+- [Grant the Agent access](#grant-the-agent-access)
+- [Install the Agent](#install-the-agent)
+- [Example Agent Configurations](#example-agent-configurations)
+- [Install the RDS integration](#install-the-rds-integration)
+- [Further reading](#further-reading)
```
Comment (cswatt) [inline content/en/database_monitoring/setup_sql_server/rds.md:30]:
```suggestion
1. [Configure the AWS integration](#configure-the-aws-integration)
1. [Grant the Agent access](#grant-the-agent-access)
1. [Install the Agent](#install-the-agent)
1. [Install the RDS integration](#install-the-rds-integration)
```

this is meant to be a numbered list of steps, not a table of contents (available in the right-hand panel)

File: content/en/database_monitoring/setup_sql_server/rds.md [hunk 5]
```diff
@@ -50,6 +53,8 @@ GO
 --Set context to msdb database and create datadog user
 USE [msdb];
 CREATE USER datadog FOR LOGIN datadog;
+-- If you intend on using log shipping monitoring (availably in agent v7.50), proceed with the following:
```
Comment (cswatt) [inline content/en/database_monitoring/setup_sql_server/rds.md:56]:
```suggestion
-- To use Log Shipping Monitoring (available in Agent v7.50+), uncomment the next line:
```

File: content/en/database_monitoring/setup_sql_server/selfhosted.md [hunk 6]
```diff
@@ -26,8 +26,11 @@ Database Monitoring provides deep visibility into your Microsoft SQL Server data
 
 Do the following steps to enable Database Monitoring with your database:
 
-1. [Grant the Agent access to the database](#grant-the-agent-access)
-2. [Install the Agent](#install-the-agent)
+- [Before you begin](#before-you-begin)
+- [Grant the Agent access](#grant-the-agent-access)
+- [Install the Agent](#install-the-agent)
+- [Example Agent Configurations](#example-agent-configurations)
+- [Further reading](#further-reading)
```
Comment (cswatt) [inline content/en/database_monitoring/setup_sql_server/selfhosted.md:33]:
```suggestion
1. [Grant the Agent access](#grant-the-agent-access)
1. [Install the Agent](#install-the-agent)
```

again, this is meant to be explicit steps, not a table of contents

File: content/en/database_monitoring/setup_sql_server/selfhosted.md [hunk 7]
```diff
@@ -51,6 +54,10 @@ CREATE USER datadog FOR LOGIN datadog;
 GRANT CONNECT ANY DATABASE to datadog;
 GRANT VIEW SERVER STATE to datadog;
 GRANT VIEW ANY DEFINITION to datadog;
+-- If you intend on using monitoring log shipping (availably in agent v7.50+), proceed with the following:
```
Comment (cswatt) [inline content/en/database_monitoring/setup_sql_server/selfhosted.md:57]:
```suggestion
-- To use Log Shipping Monitoring (available in Agent v7.50+), uncomment the next three lines:
```

File: content/en/database_monitoring/setup_sql_server/selfhosted.md [hunk 8]
```diff
@@ -60,6 +67,10 @@ CREATE LOGIN datadog WITH PASSWORD = '<PASSWORD>';
 CREATE USER datadog FOR LOGIN datadog;
 GRANT VIEW SERVER STATE to datadog;
 GRANT VIEW ANY DEFINITION to datadog;
+-- If you intend on using monitoring log shipping (availably in agent v7.50+), proceed with the following:
```
Comment (cswatt) [inline content/en/database_monitoring/setup_sql_server/selfhosted.md:70]:
```suggestion
-- To use Log Shipping Monitoring (available in Agent v7.50+), uncomment the next three lines:
```

**Review by cswatt — CHANGES_REQUESTED** [review]
Left suggestions.

I'm also adding a `do not merge` label to this PR. Please let the documentation team know when this feature is available and we'll merge! Thanks!

**Comment by joelmarcotte** [comment]
The feature is now available @cswatt so this is good to merge.

**Review by cswatt — APPROVED** [review]

**Pull request merged**